sdk/js/src/Client.ts

```diff
@@ -610,6 +618,18 @@ export class DevCycleClient<
         return this._closing
     }
 
+    /**
+     * Method to be called by the Isomorphic SDKs to update the bootstrapped config and user data when the server's
+     * representation has changed.
+     * NOTE: It is not recommended to call this yourself.
+     * @param config
+     * @param user
+     */
+    synchronizeBootstrapData(config: BucketedUserConfig, user: DevCycleUser) {
```

Comment:
adding this because once the client has rendered the first time, subsequent refresh + rerenders of server content will not replace the existing bootstrapped config if the server has found a new one. This method is called each time the serialized context the server is passing to the client is changed, so that the client gets updated with the new config and user that the server was using during its last rendering pass